lib/3scale_toolbox/commands/service_command/service_create.rb

```diff
@@ -0,0 +1,33 @@
+module ThreeScaleToolbox
+  module Commands
+    module ServiceCommand
+      class ServiceCreateSubcommand < Cri::CommandRunner
```

Comment:
In the old CLI the 'create' command had the following flags:

```
    -s, --service <service_id>        Specify service id // Although seems to not be used in the command even if you specify it
    -c, --serviceName <service_name>  Specify service name
    -a, --authentication <mode>       Specify authentication mode ('1' for API key, '2' for App Id / App Key, 'oauth' for OAuth mode, 'oidc' for OpenID Connect)
```

I changed the -c flag for a positional parameter because it is mandatory to specify at least the service name.
Is the 'authentication' flag used nowadays? should we add it?

Reply:
I would allow as optional params: 
* system name (default `service_name.downcase.gsub(/[^\w]/, '_')`)
* authentication: `backend_version` attr. (default: '1' for apiKey). 
* gateway: `deployment_mode` attr. (Default: 'hosted')

Reply:
:+1: